### DISCUSSION TITLE
Why image based captchas aren't supported?

### DISCUSSION THREAD

**Discussion by uzair004** [berstend/puppeteer-extra#695] [Q&A] [answered]
I can see in FAQ's section of [puppeteer ReCaptcha plugin](https://github.com/berstend/puppeteer-extra/tree/master/packages/puppeteer-extra-plugin-recaptcha) ordinary image-based captchas aren't supported. 

Why the such a common problem left behind? 2captcha don't even provide NodeJS based API

**Comment by berstend** [comment, marked as answer]
I responded in #696 :)
